dpgen2/conf/__init__.py

```diff
@@ -1,12 +1,6 @@
-from .conf_generator import (
-    ConfGenerator,
-)
-from .alloy_conf import (
-    AlloyConfGenerator,
-)
-from .file_conf import (
-    FileConfGenerator,
-)
```

Comment:
Is this way preferred? It can be set by [`force-grid-wrap`](https://pycqa.github.io/isort/docs/configuration/options.html#force-grid-wrap), but it applies to all imports.

Reply:
I would prefer the style because it leads to minimal changes when we add/delete imports.